lib/ber/writer.js

```diff
@@ -161,7 +161,7 @@ Writer.prototype.writeBuffer = function (buf, tag) {
 
 
 Writer.prototype.writeStringArray = function (strings) {
-  if ((!strings instanceof Array))
+  if (!(strings instanceof Array))
```

Comment:
I wonder if this should be updated to:

```suggestion
  if (Array.isArray(strings) === false)
```